Sming/Components/crypto/include/Crypto/HashContext.h

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include <axtls-8266/crypto/crypto.h>
+#include <WString.h>
+#include <Data/HexString.h>
+
+namespace Crypto
+{
+/**
+ * @brief
+ */
+struct Blob {
```

Comment:
This would have been a perfect opportunity to use `std::span`. Unfortunately, it's only available in c++20.